packages/ember-templates/lib/index.js

```diff
@@ -21,6 +29,39 @@ Ember.TextField = TextField;
 Ember.TextArea = TextArea;
 Ember.LinkComponent = LinkComponent;
 
+if (Ember.HTMLBars) {
+  Ember.HTMLBars.SafeString = SafeString;
```

Comment:
Can you move these `Ember.HTMLBars` exports to `ember-templates/compat` (we already normalize and ensure `Ember.HTMLBars` is present there)?